src/lib/eval.py

```diff
@@ -1,6 +1,6 @@
 class CheckBase(object):
     def __init__(self):
-        pass
+        raise NotImplementedError
```

Comment:
This is definitely not what you want. It also likely means that your subclass is incorrectly not calling the superclass __init__  We'll talk out-of-band about Python inheritance :)